src/AbstractConnection.js

```diff
@@ -121,10 +133,14 @@ export default class AbstractConnection extends EventEmitter {
    * - DATA everytime you get data from the server
    * - ERROR, ABORT, _or_ COMPLETE after the Connection is closed
    */
-  open() {}
+  open() {
+    this.opened = Date.now();
```

Comment:
This requires the extending connection to call `super()` right?

Reply:
either calling super or copying the line, yes. 

not sure if we need these metrics at all tho…